main/src/main/scala/sbt/internal/librarymanagement/AltLibraryManagementCodec.scala

```diff
@@ -0,0 +1,113 @@
+package sbt.internal.librarymanagement
```

Comment:
`sbt.internal.librarymanagement` package should be used exclusively by sbt/librarymanagement.
Mostly so we know where things are, but there might be JDK 9 Jigsaw rule that enforces one jar - one package (?). In any case this might be a good timing to move this code to sbt/librarymanagement. If not it should be under `sbt.internal`.